### PR TITLE
feat: add fields to BigPanda alert

### PR DESCRIPTION
**Pull request by rhajek**
This PR improves BigPanda alerting:
-  by default BP Alert contains all field values and tags
-  BigPanda primary and secondary property is configurable in tick script or in topic handler

```tick
alert
  .bigPanda()
  .appKey('fc39458f98e91eb0310258c3b725d643')
  .primaryProperty('device')
  .secondaryProperty('sensor_name')
  .topic('bigpanda-topic')
```
or in topic hadler
```yaml
id: bigpanda-cpu-alert
topic: bigpanda-topic
kind: bigpanda
options:
  app-key: '8f5795cada387b9f310f41fc369c88e4'
  primary-property: 'host'
  secondary-property: 'check'
  url: 'http://localhost:8080/data/v2/alerts'
```

###### Required for all non-trivial PRs
- [ ] Rebased/mergable
- [ ] Tests pass
- [ ] CHANGELOG.md updated
- [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

###### Required only if applicable
_You can erase any checkboxes below this note if they are not applicable to your Pull Request._
- [ ] [TICKscript Spec](https://github.com/influxdata/kapacitor/blob/master/tick/TICKscript.md) updated

### CODE REVIEW

**Review by docmerlin — APPROVED** [review]

File: services/bigpanda/config.go [hunk 1]
```diff
@@ -29,17 +29,20 @@ type Config struct {
...
+	// BigPanda Alert webhook URL,
+	// https://api.bigpanda.io/data/v2/alerts
 	URL string `toml:"url" override:"url"`
 }
 
 func NewConfig() Config {
-	return Config{
-		URL: defaultBigPandaAlertApi,
-	}
+	return Config{}
```
Comment (docmerlin) [inline services/bigpanda/config.go:38]: Why this change?

Comment (rhajek) [reply services/bigpanda/config.go:38]: I think that the dafault url https://api.bigpanda.io/data/v2/alerts should not be hardcoded. For now we use the generic REST API, but we want BigPanda to add our integration among the supported official integrations. Instead of a generic REST API url, we will get a special webhook url like https://inbound.bigpanda.io/influxdata/alerts, which will be generated in the bigpanda admin console.

Comment (docmerlin) [reply services/bigpanda/config.go:38]: Reasonable defaults are a good idea for users, this was a specific ask, for a previous PR.

File: services/bigpanda/service.go [hunk 2]
```diff
@@ -215,24 +218,45 @@ func (s *Service) preparePost(appKey, id string, message string, details string,
...
 		return nil, err
 	}
 
-	alertUrl, err := url.Parse(c.URL)
+	var bpUrl string
+	if hc.URL != "" {
+		bpUrl = hc.URL
+	} else {
+		bpUrl = c.URL
+	}
```
Comment (docmerlin) [inline services/bigpanda/service.go:258]: Since strings in go are immutable, we can do this which is a bit cleaner.

```go
    bpUrl := hc.URL
    if bpUrl == "" {
        bpUrl = c.URL
}
```

**Review by docmerlin — APPROVED** [review]
